lib/Runtime/Debug/DebugContext.h

```diff
@@ -37,6 +37,9 @@ namespace Js
         // The debugger is running which means that the engine is
         // running in interpreted mode.
         Debugging,
+
+        // The debugger is running but recording is suspended.
+        RecordingSuspended,
```

Comment:
Further convert that bool to a bit mask

---
In reply to: [129982675](https://github.com/Microsoft/ChakraCore/pull/3447#discussion_r129982675) [](ancestors = 129982675)